sass/blocks/_blocks.scss

```diff
@@ -296,9 +301,15 @@
 		cite {
 			display: inline-block;
 			font-family: $font__heading;
-			font-size: $font__size-xs;
 			line-height: 1.6;
 			text-transform: none;
+			color: $color__text-light;
+			
+			/* 
+			 * This requires a rem-based font size calculation instead of our normal em-based one, 
+			 * because the cite tag sometimes gets wrapped in a p tag. This is equivalent to $font-size_xs. 
+			 */
+			font-size: calc(1rem / (1.25 * #{$font__size-ratio}));
```

Comment:
Since this is just a solution for fallback scenarios, I think this will be fine. Ideally this would benefit from the solution proposed in #164, and we can always revisit this as a part of that PR.